src/common/enums.js

```diff
@@ -141,22 +141,26 @@ export const fileTypeDetail = {
   SHM: {
     icon: 'shipping',
     title: 'Shipping Manifest',
-    description: 'File type description goes here...',
+    description:
+      'Manifest of biospecimens shipped to the sequencing center. The biospecimen identifiers from this file will be used to link all clinical and genomic data in the KFDRC platform.',
   },
   CLN: {
     icon: 'hospital',
-    title: 'Clinical/Phenotype Data',
-    description: 'File type description goes here...',
+    title: 'Clinical/Phenotypic Data',
+    description:
+      'Structured files with data linked to the biospecimen identifiers provided in the shipping manifest(s). Exports from data management applications, such as REDCap, are the fastest to process, but we will work with you on most formats provided as long as a description is provided.',
   },
   SEQ: {
     icon: 'dna',
     title: 'Sequencing Manifest',
```

Comment:
Does this include S3 file manifests? They don't normally contain the submitter's ID for the specimens, but they do contain the KF IDs for the specimens, put there by the bix team. Should we have a separate category for this?

Reply:
🤔 this doesn't include S3 manifest, for now it's meant to be the manifest we receive from the sequencing center. 

If we did make a Storage Manifest maybe we just hide it from the non Admins since it's an internal doc?